packages/scene-composer/src/utils/dataBindingTemplateUtils.ts

```diff
@@ -96,3 +96,14 @@ export function applyDataBindingTemplate(
   });
   return dataBindingContext;
 }
+
+/**
+ * We extract entityId from the data binding object.
+ * @param dataBinding we send data binding object
+ * @returns
+ */
+export const extractEntityId = (dataBinding: IValueDataBinding): string => {
+  const contextData = dataBinding.dataBindingContext ?? {};
+  const bindingKeys = Object.keys(contextData);
+  return contextData[bindingKeys[0]];
```

Comment:
Since we can't control the order that the keys are in a binding and they can have more than just 1 key (like component, property we might might to do something more like https://github.com/awslabs/iot-app-kit/blob/main/packages/scene-composer/src/utils/dataBindingUtils.ts#L38 so it's more reliable and be reused with tags etc.